proposals/simd/portable-simd.md

```diff
@@ -0,0 +1,1324 @@
+| `v16x8` |         16 |       8 | `b16x8`    |
+| `v32x4` |         32 |       4 | `b32x4`    |
+| `v64x2` |         64 |       2 | `b64x2`    |
+
+
+### Modulo integer interpretations
+
+The bits in a lane can be interpreted as integers with modulo arithmetic
+semantics. Many arithmetic operations can be defined on these types which don't
+impose a signed or unsigned integer interpretation.
+
+* `i8x16 : v8x16`: Each lane is an `i8`.
+* `i16x8 : v16x8`: Each lane is an `i16`.
+* `i32x4 : v32x4`: Each lane is an `i32`.
+* `i64x2 : v64x2`: Each lane is an `i64`.
+
+Additional properties:
+
+|    S    | S.LaneType |
+|---------|------------|
+| `i8x16` |       `i8` |
+| `i16x8` |      `i16` |
+| `i32x4` |      `i32` |
+| `i64x2` |      `i64` |
+
+### Signed integer interpretations
+
+Each lane is interpreted as a two's complement integer.
+
+* `s8x16 : i8x16`: Lane values in the range -2^7 – 2^7-1.
+* `s16x8 : i16x8`: Lane values in the range -2^15 – 2^15-1.
+* `s32x4 : i32x4`: Lane values in the range -2^31 – 2^31-1.
+* `s64x2 : i64x2`: Lane values in the range -2^63 – 2^63-1.
+
+These interpretations get additional properties defining the range of values in
+a lane:
+
+|    S    | S.Min | S.Max  |
+|---------|------:|-------:|
+| `s8x16` | -2^7  | 2^7-1  |
+| `s16x8` | -2^15 | 2^15-1 |
+| `s32x4` | -2^31 | 2^31-1 |
+| `s64x2` | -2^63 | 2^63-1 |
+
+### Unsigned integer interpretations
+
+Each lane is interpreted as an unsigned integer.
+
+* `u8x16 : i8x16`: Lane values in the range 0 – 2^8-1.
+* `u16x8 : i16x8`: Lane values in the range 0 – 2^16-1.
+* `u32x4 : i32x4`: Lane values in the range 0 – 2^32-1.
+* `u64x2 : i64x2`: Lane values in the range 0 – 2^64-1.
+
+These interpretations get additional properties defining the range of values in
+a lane:
+
+|    S    | S.Min | S.Max  |
+|---------|------:|-------:|
+| `u8x16` |     0 | 2^8-1  |
+| `u16x8` |     0 | 2^16-1 |
+| `u32x4` |     0 | 2^32-1 |
+| `u64x2` |     0 | 2^64-1 |
+
+### Floating-point interpretations
+
+Each lane is interpreted as an IEEE floating-point number.
+
+* `f32x4 : v32x4`: Each lane is an `f32`.
+* `f64x2 : v64x2`: Each lane is an `f64`.
+
+Additional properties:
+
+|    S    | S.LaneType |
+|---------|------------|
+| `f32x4` | `f32`      |
+| `f64x2` | `f64`      |
+
+# Floating-point semantics
+
+The floating-point operations in this specification aim to be conforming to
+[IEEE 754-2008][ieee754] while being compatible with WebAssembly and JavaScript.
+Some things which are left unspecified by the IEEE standard are given stricter
+semantics by WebAssembly.
+
+## Rounding modes
+
+Floating-point operations that need a *rounding mode* take a `RoundingMode`
+operand which provides the rounding mode to use for the operation.
+
+* `TiesToEven`: Round to nearest, ties towards even.
+* `TowardPositive`: Round towards positive infinity.
+* `TowardNegative`: Round towards negative infinity.
+* `TowardZero`: Round towards zero.
+
+## Default NaN value
```

Comment:
To confirm: the intent here is to precisely match [the latitude which WebAssembly provides](https://github.com/WebAssembly/design/blob/master/Semantics.md#floating-point-operators) w.r.t. NaNs, correct? I find it odd to repeat it here, with different wording.